packages/split-view/README.md

```diff
@@ -1,43 +1,79 @@
-## Description
+## Overview
 
-An `sp-split-view` element delivers its first two direct child elements in a horizontal or vertical (`<sp-split-view vertical>`) orientation that distributes the available page real estate as per the supplied attribute API. When leveraging the resizable attribute a pointer and keyboard accessible affordance is provided for the user to customize the distribution of that area between the available children.
+An `sp-split-view` element displays its first two direct child elements side by side (horizontal) or stacked (vertical with `vertical` attribute). The component automatically distributes the available space between the two panels. When the `resizable` attribute is added, users can drag the splitter or use keyboard controls to adjust the panel sizes.
 
 ### Usage
 
 [![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/split-view?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/split-view)
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/split-view?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/split-view)
 [![Try it on Stackblitz](https://img.shields.io/badge/Try%20it%20on-Stackblitz-blue?style=for-the-badge)](https://stackblitz.com/edit/vitejs-vite-2ilo6nmh)
 
-```
+```zsh
 yarn add @spectrum-web-components/split-view
 ```
 
 Import the side effectful registration of `<sp-split-view>` via:
 
-```
+```js
 import '@spectrum-web-components/split-view/sp-split-view.js';
 ```
 
 When looking to leverage the `SplitView` base class as a type and/or for extension purposes, do so via:
 
-```
+```js
 import { SplitView } from '@spectrum-web-components/split-view';
 ```
 
-## Variants
+### Options
+
+#### Collapsible
+
+Use the `collapsible` attribute to allow the user to collapse the split view.
+
+#### Label
+
+Use the `label` attribute to set the `aria-lavel` on the `splitter` element.
+
+#### Primary Max
+
+`primary-max` is the maximum size of the primary pane.
+
+#### Primary Min
+
+`primary-min` is the minimum size of the primary pane.
 
-### Horizontal
+#### Secondary Max
 
-```html
+`secondary-max` is the maximum size of the secondary pane.
+
+#### Secondary Min
+
+`secondary-min` is the minimum size of the secondary pane.
```

Comment:
This can stay as is, but you could also consider grouping the min/maxes here since the multiple headings with one line of text are a bit repetitive.